packages/headless/src/controllers/ssr/search-box/headless-ssr-search-box.ts

```diff
@@ -0,0 +1,23 @@
+import {SearchEngine} from '../../../app/search-engine/search-engine';
+import {ControllerDefinitionWithoutProps} from '../../../app/ssr-engine/types/common';
+import {
+  SearchBox,
+  SearchBoxProps,
+  buildSearchBox,
+} from '../../search-box/headless-search-box';
+
+export type {
```

Comment:
From this file, I prefer not to, since even in `controllers/index.ts` we don't tend to export `*`, therefore we don't know for sure that we want to export everything here.

Additionally, some controllers (such as search parameters manager) won't need some types (such as `*Props`).